### PR TITLE
[BEE-35523] Update K8s GKE example installations

### PR DESCRIPTION
**Pull request by jsmith-cb**
Jira: [BEE-35523](https://cloudbees.atlassian.net/browse/BEE-35523)

Previews:
* [Intro page](https://github.com/cloudbees/cloudbees-examples/tree/BEE-35523/cloudbees-cd/kubernetes/gke)
* [Demo page](https://github.com/cloudbees/cloudbees-examples/blob/BEE-35523/cloudbees-cd/kubernetes/gke/demo.md)
* [Agents page](https://github.com/cloudbees/cloudbees-examples/blob/BEE-35523/cloudbees-cd/kubernetes/gke/agents.md)
* [Clustered page](https://github.com/cloudbees/cloudbees-examples/blob/BEE-35523/cloudbees-cd/kubernetes/gke/clustered.md)

[BEE-35523]: https://cloudbees.atlassian.net/browse/BEE-35523?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: cloudbees-cd/kubernetes/gke/prod.md [hunk 1]
```diff
@@ -208,60 +258,94 @@ Be sure to follow the security policies and rules of your organization.
       --set storage.volumes.serverPlugins.storageClass=filestore-sc \
```
[the remark this answers is not in the document]
Comment (jsmith-cb) [reply cloudbees-cd/kubernetes/gke/prod.md]: Updated with [5bd1de4](https://github.com/cloudbees/cloudbees-examples/pull/69/commits/5bd1de4eded1165ecdeae523cf18b707f3110068).

File: cloudbees-cd/kubernetes/gke/prod.md [hunk 2]
```diff
@@ -122,78 +160,90 @@ Be sure to follow the security policies and rules of your organization.
...
 
-- Download production values file
+Now that your cluster database is configured, you can install CloudBees CD/RO. To get started:
+
+1. Download production values file:
     ```bash
-    curl -fsSL -o cloudbees-cd-production.yaml https://raw.githubusercontent.com/cloudbees/cloudbees-examples/master/cloudbees-cd/kubernetes/cloudbees-cd-prod.yaml
-  ```
-- Create k8s namespace
+    curl -fsSL -o cloudbees-cd-demo.yaml $PROD_FILE_URL
```
[the remark this answers is not in the document]
Comment (jsmith-cb) [reply cloudbees-cd/kubernetes/gke/prod.md]: Updated with [5bd1de4](https://github.com/cloudbees/cloudbees-examples/pull/69/commits/5bd1de4eded1165ecdeae523cf18b707f3110068).

File: cloudbees-cd/kubernetes/gke/prod.md [hunk 3]
```diff
@@ -122,78 +160,90 @@ Be sure to follow the security policies and rules of your organization.
...
+3. Create the database credentials secret:
     ```bash
     kubectl create secret generic flow-db-secret \
       --namespace $NAMESPACE \
       --from-literal=DB_USER=$GCP_DB_USER \
       --from-literal=DB_PASSWORD=$GCP_DB_PASSWORD
-  ```
-- Set db connection parameters in the values file
+    ```
+4.  Open `cloudbees-cd-demo.yaml` and set the following database connection parameters:
```
[the remark this answers is not in the document]
Comment (jsmith-cb) [reply cloudbees-cd/kubernetes/gke/prod.md]: Updated with [5bd1de4](https://github.com/cloudbees/cloudbees-examples/pull/69/commits/5bd1de4eded1165ecdeae523cf18b707f3110068).

File: cloudbees-cd/kubernetes/gke/README.md [hunk 4]
```diff
@@ -1,26 +1,41 @@
...
 
-[Example of environment and installation in demo mode](demo.md)
+## GKE available examples <a name="cdro-gke-available-examples"/>
+
+The following example installations are provided: 
+
+* [GKE example CloudBees CD/RO demo installation](demo.md)
 
-[Example of environment and installation in production mode](prod.md)
+* [GKE example CloudBees CD/RO production installation](clustered.md)
```
[the remark this answers is not in the document]
Comment (jsmith-cb) [reply cloudbees-cd/kubernetes/gke/README.md:39]: Updated.

File: cloudbees-cd/kubernetes/gke/clustered.md [hunk 5]
```diff
@@ -0,0 +1,351 @@
...
+  GCP_VPC_NETWORK=<GCP-VPC-NETWORK>                     # e.g. GCP_VPC_NETWORK=cd-vpc
+  GCP_VPC_SUBNET=<GCP-VPC-SUBNET>                       # e.g. GCP_VPC_SUBNET=cd-subnet
+  GCP_SUBNET_IP_RANGE=<GCP-SUBNET-IP-RANGE>             # e.g. GCP_SUBNET_IP_RANGE=10.0.0.0/24
+  GKE_CLUSTER_NAME=<GKE-CLUSTER-NAME>                   # e.g. GKE_CLUSTER_NAME=gke-cd-clustered
+  # Number of nodes in the cluster, 3 is enough for clustering purposes
+  GKE_CLUSTER_NUM_NODES=<GKE-CLUSTER-NUMBER-OF-NODES>   # e.g. GKE_CLUSTER_NUM_NODES=3
+  # Machine type for the GKE cluster nodes, e2-standard-8 is enough for clustering purposes
+  GKE_CLUSTER_MACHINE_TYPE=<GKE-CLUSTER-MACHINE-TYPE>   # e.g. GKE_CLUSTER_MACHINE_TYPE=e2-standard-8
+  HELM_RELEASE=<CLOUDBEES-CD-HELM-RELEASE>              # e.g. HELM_RELEASE=cd-prod
+  NAMESPACE=<CLOUDBEES-CD-NAMESPACE>                    # e.g. NAMESPACE=cd-prod
```
[the remark this answers is not in the document]
Comment (jsmith-cb) [reply cloudbees-cd/kubernetes/gke/clustered.md:46]: Added.

File: cloudbees-cd/kubernetes/gke/README.md [hunk 6]
```diff
@@ -1,26 +1,41 @@
-# CloudBees CD GKE examples
 
-## In this folder
+# CloudBees CD/RO GKE example installations <a name="cdro-gke-example"/>
 
-This folder contains instructions to quickly install the CloudBees CD Helm Chart on GKE cluster.
+This directory provides example installations of CloudBees CD/RO within GKE clusters, and provide more insight to the instructions in [CloudBees CD/RO for Google Cloud Platform (GCP)](https://docs.cloudbees.com/docs/cloudbees-cd/latest/install-k8s/k8s-platform-specific-configurations#_google_cloud_platform_gcp).
```
Comment (liamfield) [inline cloudbees-cd/kubernetes/gke/README.md:4]:
```suggestion
This directory provides example installations of CloudBees CD/RO within GKE clusters, and insight into the instructions in [CloudBees CD/RO for Google Cloud Platform (GCP)](https://docs.cloudbees.com/docs/cloudbees-cd/latest/install-k8s/k8s-platform-specific-configurations#_google_cloud_platform_gcp).
```

File: cloudbees-cd/kubernetes/gke/README.md [hunk 7]
```diff
@@ -1,26 +1,41 @@
...
 
-Pre-requisites:
+>**IMPORTANT** 
+> 
+>All examples provided are for informational purposes only. They are not meant to be used in production environments, but only to provide working demonstrations of such environments.
```
Comment (liamfield) [inline cloudbees-cd/kubernetes/gke/README.md:8]:
```suggestion
>All examples provided are for informational purposes only. They are meant to provide working demonstrations of production environments, not to be used in such environments. 
```

File: cloudbees-cd/kubernetes/gke/agents.md [hunk 8]
```diff
@@ -1,41 +1,64 @@
...
+>
+> CloudBees CD/RO installation include the CloudBees CD/RO bound agent (`flow-bound-agent`), but this agent is an internal component used specifically by CloudBees CD/RO for internal operations. While it is possible to schedule user jobs on bound agents, they are not intended for this purpose, and the overall performance of CloudBees CD/RO may be greatly impacted. CloudBees CD/RO agents should be used instead.
+
+## Prerequisites
+To complete the following instructions, you must meet the cluster and tooling requirements listed in [Prerequisites](README.md#gke-available-examples-a-namecdro-gke-available-examples).
+
+
+## Configure environment variables
+
+The commands in following sections are preconfigured to use environment variables. To align your installation, set the following environment variables:
```
Comment (liamfield) [inline cloudbees-cd/kubernetes/gke/agents.md:15]:
```suggestion
The commands in the following sections are preconfigured to use environment variables. To align your installation, set the following environment variables:
```

**Review by liamfield — APPROVED** [review]
LGTM; some minor grammatical suggestions.

**Pull request merged**